migrations/versions/1bcbafcc37ee_projects.py

```diff
@@ -0,0 +1,4037 @@
+"""projects
+
+Revision ID: 1bcbafcc37ee
+Revises: 6e3b2b3e56f0
+Create Date: 2022-02-17 16:43:19.334423
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1bcbafcc37ee'
+down_revision = '6e3b2b3e56f0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    project = op.create_table('projects',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('location', sa.String(), nullable=True),
+    sa.Column('capital_investment', sa.Float(), nullable=True),
+    sa.Column('epic_guid', sa.String(), nullable=True),
+    sa.Column('is_project_closed', sa.Boolean(), nullable=True),
+    sa.Column('sub_sector_id', sa.Integer(), nullable=False),
+    sa.Column('proponent_id', sa.Integer(), nullable=False),
+    sa.Column('region_id_env', sa.Integer(), nullable=False),
+    sa.Column('region_id_flnro', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['proponent_id'], ['proponents.id'], ),
+    sa.ForeignKeyConstraint(['region_id_env'], ['regions.id'], ),
+    sa.ForeignKeyConstraint(['region_id_flnro'], ['regions.id'], ),
+    sa.ForeignKeyConstraint(['sub_sector_id'], ['sub_sectors.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.alter_column('ea_acts', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('federal_involvements', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('ministries', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('proponents', 'is_active',
+               existing_type=sa.BOOLEAN(),
+               nullable=True)
+    op.alter_column('proponents', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('regions', 'entity',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('regions', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('sectors', 'short_name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('sectors', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('sub_sectors', 'short_name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('sub_sectors', 'name',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.bulk_insert(project, [
+	{
```

Comment:
Hmm, why are we inserting projects as part of migration as project is transactional data?
If it is just for testing purpose; better to add it as a script or make sure you add another migration where you delete all these projects.

Reply:
Removed it